yarn.lock

```diff
@@ -1131,6 +1131,22 @@
     humps "^2.0.1"
     prop-types "^15.5.10"
 
+"@glass/domainproperties@0.0.49":
```

Comment:
This needs to be fixed. Need to make sure to bump the querygrid/package.json at the same time that domainproperties is bumped.

Reply:
Yes, thanks for the reminder.  I bumped and rebuilt and domainproperties is no longer in yarn.lock.